images/installer-artifacts/Dockerfile.rhel

```diff
@@ -4,8 +4,8 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
 WORKDIR /go/src/github.com/openshift/installer
 COPY . .
-RUN go generate ./data && \
-    SKIP_GENERATION=y GOOS=darwin GOARCH=amd64 hack/build.sh
+RUN make cross-build-darwin-amd64 cross-build-linux-amd64
```

Comment:
Why do you need to build for Linux here?  Don't we get that via the `FROM ...:installer` line below?

Reply:
We want to have a linux/amd64 installer even in the other architecture's payloads.  For remote deployments, just like we enable installing from mac regardless of cluster architecture, we also want to enable doing such from Linux on commodity hardware, which is still most commonly x86.